.drone.yml

```diff
@@ -19,39 +19,48 @@ pipeline:
     when:
       event: [push, pull_request]
 
-  npm:  # old `npm install`
-    image: node:slim
+  nodejs-build:  # with `npm install` inside
+    image: docker/compose:1.22.0
     environment:
       - DEPS_DIR=/drone/deps
     commands:
-      - npm install
-      - npm install -g gulp-cli
-      - gulp build
+      - cd docker/
+      - cp drone_env/.env . && cp drone_env/* env_files/
+      # nodejs build is not in `docker-build` section,
+      # because we have this deps:
+      # build nodejs -> gulp build -> build python-prod
+      - docker-compose -f docker-compose-build.yml build nodejs
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
     when:
       event: [push, pull_request]
 
-# @todo #352:30m Fix volumes collision for new drone `npm install`.
-#  `docker-compose run --rm nodejs gulp build` don't create
-#  build dir at host folder.
-#  When collision is resolved, change old `npm install` for new one.
+  # push nodejs image with separated step, because docker-compose image in npm install
+  # doesn't provide `docker` command
+  nodejs-push:
```

Comment:
I do not understand, why do you push it here and not in `docker-push`?

Reply:
@artemiy312 , see comment at the line befor `nodejs-push`. It's not clear?